tests/PHPStan/Analyser/data/array-types.php

```diff
@@ -0,0 +1,31 @@
+<?php
+
+namespace NonEmptyArray;
+
+use function PHPStan\Testing\assertType;
+
+class Foo
+{
+
+	/**
+	 * @param array{foo: '1', bar: '2', lall: '3'} $array1
+	 * @param array{foo: '1', bar: '4', lall2: '3'} $array2
+	 */
+	public function arrayFunctions($array1, $array2): void
+	{
+		assertType("array('1' => '1', '2' => '2', '3' => '3')", array_combine($array1, $array1));
```

Comment:
This combine asserts are unrelated?

Reply:
Yes it's unrelated. I only copy&past some tests here so that I can see the output from phpstan.